package.json

```diff
@@ -28,6 +29,7 @@
   },
   "homepage": "https://github.com/abahachuk/ptushki-backend#readme",
   "dependencies": {
+    "@types/nodemailer": "^6.2.0",
     "@types/supertest": "^2.0.7",
```

Comment:
Could you move these types to dev dependencies?

Reply:
done